gammapy/astro/darkmatter/spectra.py

```diff
@@ -325,21 +330,27 @@ class DarkMatterDecaySpectralModel(SpectralModel):
     )
     scale._is_norm = True
 
+    dfactor = Parameter(
+        "dfactor",
+        1,
+        unit="GeV cm-2",
+        interp="log",
+    )
+
     tag = ["DarkMatterDecaySpectralModel", "dm-decay"]
 
-    def __init__(self, mass, channel, scale=scale.quantity, jfactor=1, z=0):
+    def __init__(self, mass, channel, scale=scale.quantity, dfactor=dfactor, z=0):
```

Comment:
the mass could be a parameter frozen by default, but it could be done in a separate PR.